### PR TITLE
Pester tests assertion update

### PR DESCRIPTION
**Pull request by timayabi2020**
﻿

### Changes proposed in this pull request

- Update pester test assertion for module imports that write to streams when debug preference is set to 'Inquire'. The streams debug count returns a value of 2 yet the expected value is 0.
image

### CODE REVIEW

File: tools/TestModule.ps1 [hunk 1]
```diff
@@ -35,6 +36,6 @@ $PesterConfiguration.TestResult.Enabled = $true
 $PesterConfiguration.TestResult.OutputPath = (Join-Path $ModuleTestsPath "$moduleName-TestResults.xml")
 
 $TestResults = Invoke-Pester -Configuration $PesterConfiguration
-If ($TestResults.FailedCount -gt 0) { Write-Error "$($TestResults.FailedCount) tests failed." }
```
Comment (peombwa) [inline tools/TestModule.ps1:38]: @timayabi2020, why are we changing this to 5? We shouldn't have filing tests.

File: tools/TestModule.ps1 [hunk 2]
```diff
@@ -16,6 +16,7 @@ if(-not $Isolated) {
 }
 
 $modulePsd1 = Get-Item -Path (Join-Path $ModulePath "./$ModuleName.psd1")
+Write-Host $modulePsd1
```
Comment (peombwa) [inline tools/TestModule.ps1:19]: Let's remove the log.

File: tools/TestModule.ps1 (hunk 1 above)
Comment (timayabi2020) [reply tools/TestModule.ps1]: Thank you for flagging this. It was an error from my side. This has been changed

File: tools/TestModule.ps1 (hunk 2 above)
Comment (timayabi2020) [reply tools/TestModule.ps1]: This has been removed

**Review by peombwa — APPROVED** [review]

**Pull request merged**